terraform/modules/monitoring/alarms.tf

```diff
@@ -77,3 +77,65 @@ resource "aws_cloudwatch_metric_alarm" "sidekiq_memory" {
     ServiceName = var.sidekiq_service_name
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "rds_cpu" {
+  alarm_actions             = [aws_sns_topic.this.arn]
+  alarm_name                = "${var.prefix}-rds-cpu"
+  comparison_operator       = "GreaterThanOrEqualToThreshold"
+  datapoints_to_alarm       = 3
+  evaluation_periods        = 5
+  metric_name               = "CPUUtilization"
+  namespace                 = "AWS/RDS"
+  ok_actions                = [aws_sns_topic.this.arn]
+  period                    = 60
+  statistic                 = "Average"
+  threshold                 = 90
+  insufficient_data_actions = []
+
+  dimensions = {
+    DBInstanceIdentifier = var.database_id
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "rds_storage" {
+  alarm_actions             = [aws_sns_topic.this.arn]
+  alarm_name                = "${var.prefix}-rds-storage"
+  comparison_operator       = "LessThanOrEqualToThreshold"
+  datapoints_to_alarm       = 1
+  evaluation_periods        = 1
+  ok_actions                = [aws_sns_topic.this.arn]
+  threshold                 = 15
+  insufficient_data_actions = []
+
+  metric_query {
+    id = "freeStorageSpacePercentage"
+    expression  = "freeStorageSpace/${var.database_allocated_storage}"
+    period      = 300
+    return_data = "true"
+  }
+
+  metric_query {
+    id = "freeStorageSpace"
+
+    metric {
+      metric_name = "FreeStorageSpace"
+      namespace   = "AWS/RDS"
+      period      = 300
+      stat        = "Minimum"
```

Comment:
I'm assuming storage space isn't really volatile over very short timescales, so I went for minimum here as the stat. If for some reason we're expecting the storage space to quickly fluctuate as part of how the db functions, we should probably change this to average or something.